docker/dockerfile_build.bzl

```diff
@@ -0,0 +1,113 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Lets move this rule to contrib,
to indicate its still experimental

Reply:
Done